### ISSUE TITLE
[B005][Backend] As a User, I should be able to view the Data for each scraped word

### ISSUE THREAD

**Issue by AetheryNA** [AetheryNA/google_scraper_ruby#10]
## Why

The user should be able to view all the data scraped from google and stored under their profile. This way the user can see the data they wanted from the uploaded CSV.

## Acceptance Criteria

- Fetch all the data stored for each keyword

**Comment by carryall** [comment]
@AetheryNA Do we need another issue to work on the UI part for the scraped data detail? e.g. showing the number of ad links and also a list of links and the page cache

**Comment by AetheryNA** [comment]
Yup agreed! Created it [here](https://github.com/AetheryNA/google_scraper_ruby/issues/16) 😸
